metricbeat/module/munin/munin.go

```diff
@@ -36,6 +37,12 @@ const (
 	unknownValue = "U"
 )
 
+var (
+	// Field names must match with this expression
+	// http://guide.munin-monitoring.org/en/latest/reference/plugin.html#notes-on-fieldnames
```

Comment:
Do you know if munin only specifies this or also checks for it?

Reply:
According to the linked documentation, fields that don't match this regexp shouldn't be allowed.

Now, I have tried with munin implementation and I have seen that fields containing dots are actually rejected, but other characters are just replaced by underscores and its metrics accepted.

I think that for dots we should also reject them, and for the rest of characters we can add an option so the user decides if they want to replace other characters mimicking munin, or leave them as they are.